src/AltinnCore/Runtime/Dockerfile

```diff
@@ -15,8 +14,8 @@ RUN dotnet publish AltinnCore.Runtime.csproj -c Release -o /app_output
 # Altinn-Studio runtime app
 FROM altinn-runtime-app:latest AS generate-runtime-app
 
-#dotnet:2.2-sdk
-FROM microsoft/dotnet@sha256:7d8256eead49252ac2de7079268659102f44a6e40e7890fec2a7633d0b374470 AS final
+#dotnet:2.2-sdk - TODO: Investigate why we're using SDK here
```

Comment:
We're using SDK here because we're re-using this Runtime image when we're building the App.